File: 3st/Juram/programmer/삼각형의 완성조건 (1).js
function solution(sides) {
    let max = 0;
    let sum = 0;
    let idx = 0;
    for(let i=0; i<sides.length; i++){
        if(sides[i]>max){
            max = sides[i];
            idx = i;
        }
    }
    for(let i=0; i<sides.length; i++){
        if(idx == i) continue;
        sum += sides[i];
    }
    if(sum<=max) return 2;
    else return 1;
}

File: 4st/Jisoo/programmer/삼각형의 완성조건 (1).java
class Solution {
    public int solution(int[] sides) {
        int answer = 0;
       int max=Math.max(sides[0],Math.max(sides[1],sides[2]));
        int sum=sides[0]+sides[1]+sides[2]-max;
            if(max<sum){
                return 1;
            }
            else{
                return 2;
            }
    }
}